app/src/main/java/com/smaillimp/yatzy/feature_player/use_case/ValidatePlayerName.kt

```diff
@@ -7,6 +7,16 @@ class ValidatePlayerName {
                 successful = false,
                 errorMessage = "Please provide a user name."
             )
+        } else if (playerName.length > 15) {
+            return ValidationResult(
+                successful = false,
+                errorMessage = "Player name is too long."
+            )
+        } else if (playerName.length == 1) {
```

Comment:
I think it could be a bit more elegant by using when (see link below). It would avoid having repetitive if... else if... else if statements. 

https://stackoverflow.com/questions/53138172/how-to-implement-switch-case-statement-in-kotlin#53138234

Reply:
Will do 👍